terraso_backend/tests/core/models/test_groups.py

```diff
@@ -20,6 +22,23 @@ def test_group_is_slugifiable_by_name():
     assert group.slug == "this-is-my-name"
 
 
+def test_group_disallowed_name():
+    group_name = "New"
+    group = mixer.blend(Group, name=group_name)
+
+    with pytest.raises(ValidationError, match="New is not allowed as a name"):
+        group.full_clean()
+
+
+def test_group_additional_disallowed_name():
```

Comment:
The correct way to tweak settings in a test is receiving `settings` as fixture.

https://pytest-django.readthedocs.io/en/latest/helpers.html#settings

Reply:
@cacarrara Can you make the change use the GitHub suggest feature?